src/ppc64.h

```diff
@@ -52,4 +52,55 @@
 #define BF_SCALE			0
 #define BF_X2				0
 
+#define SIMD_COEF_32		4
+#define SIMD_COEF_64		2
+
+#ifndef SIMD_PARA_MD4
+#define SIMD_PARA_MD4		1
+#endif
+#ifndef SIMD_PARA_MD5
+#define SIMD_PARA_MD5		1
+#endif
+#ifndef SIMD_PARA_SHA1
+#define SIMD_PARA_SHA1		1
+#endif
+#ifndef SIMD_PARA_SHA256
+#define SIMD_PARA_SHA256	1
+#endif
+#ifndef SIMD_PARA_SHA512
+#define SIMD_PARA_SHA512	1
+#endif
+
+#define STR_VALUE(arg)			#arg
```

Comment:
I think the code from line 74 to line 102 should be somehow reused for different archs, instead of being copied & pasted whenever a new arch is supported.